superset-frontend/packages/superset-ui-core/src/style/index.tsx

```diff
@@ -160,6 +160,7 @@ const defaultTheme = {
   },
   transitionTiming: 0.3,
   gridUnit: 4,
+  transparent: 'rgba(255, 255, 255, 0)',
```

Comment:
Heads up to @michael-s-molina @geido that we might want to consider this in our Theme discussions. We might want to consider a means to add "opacity stops" that we support for some use cases, and a means to use them as alpha channel values on any given theme color, or perhaps as a styling mixin of some sort. The reason being that this approach works great while we're in light/white theming mode, but if we switched to dark mode or something, then we'd again see weird grays as out gradient transitions toward transparent white.

Reply:
@stephenLYZ I think we can allow this for now, but if you can think of a way to generalize alpha channel transparency for colors in a more generalized way, I think we'd be interested :)

Reply:
Yeah. The way I've come up with is to convert the theme hex color to rgba via an external method, such as `emotion-rgba`:

```
rgba(theme.colors.grayscale.light5, 0)
```

Reply:
Seems reasonable/extendable to me! Cool!